app.rb

```diff
@@ -0,0 +1,174 @@
+$LOAD_PATH << '.'
+require './person'
+require './student'
+require './teacher'
+require './classroom'
+require './book'
+require './rental'
+
+class App
+  def initialize
+    @people = []
+    @books = []
+    @rentals = []
+    @class = Classroom.new('Grade 5')
+  end
+
+  def run
+    print 'Welcome to School Library App'
+    sleep 0.75
+    menu
+  end
+
+  def menu
+    puts
+    puts 'Please choose an option by entering a number'
+    puts '1 - List all books'
+    puts '2 - List all people'
+    puts '3 - Create a person'
+    puts '4 - Create a book'
+    puts '5 - Create a rental'
+    puts '6 - List all rentals for a given person id'
+    puts '7 - Exit'
+    option = gets.chomp
+
+    get_option option
+  end
+
+  def get_option(input) # rubocop:todo Metrics/CyclomaticComplexity
+    case input
+    when '1'
+      list_all_books
+    when '2'
+      list_all_people
+    when '3'
+      create_a_person
+    when '4'
+      create_a_book
+    when '5'
+      create_a_rental
+    when '6'
+      list_rentals_by_person_id
+    when '7'
+      puts 'Thank you for using this app!'
+    else
+      puts 'Please enter a number between 1 and 7'
+    end
+  end
+
+  def list_all_books
+    puts 'There are no books yet! Kindly add books.' if @books.empty?
+
+    @books.each { |book| puts "Title: #{book.title}, Author: #{book.author}" }
+    sleep 0.75
+    menu
+  end
+
+  def list_all_people
+    puts 'There are no people yet! Kindly add a student or teacher.' if @people.empty?
+    @people.map { |person| puts "[#{person.class}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}" }
+    sleep 0.75
+    menu
+  end
+
+  def create_a_person
+    print 'Do you want to create a student (1) or teacher (2) [Input a number]: '
+    option = gets.chomp
+
+    case option
+    when '1'
+      create_a_student
+    when '2'
+      create_a_teacher
+    else
+      puts 'Invalid input. Kindly type 1 or 2'
+    end
+  end
```

Comment:
- [ ] I'm unable to create a person(Student or Teacher). 
- Screenshot: 
![Screenshot from 2022-06-16 12-16-02](https://user-images.githubusercontent.com/63087491/174058809-3bc59ca9-8324-4448-8d4a-e022b60b3095.png)